src/modules/routing/messages/RoutingMessageType.ts

```diff
@@ -8,4 +8,5 @@ export enum RoutingMessageType {
   BatchPickup = 'https://didcomm.org/messagepickup/1.0/batch-pickup',
   Batch = 'https://didcomm.org/messagepickup/1.0/batch',
   ForwardMessage = 'https://didcomm.org/routing/1.0/forward',
+  RequestMediation = 'https://didcomm.org/coordinate-mediation/1.0/mediate-request',
```

Comment:
that it is and should be mediation-request, not mediate-request. I will correct this.
We use `https://didcomm.org/coordinate-mediation`, but I have not seen coordinate-mediation at didcomm.org. Is that ok?

Reply:
I'm not sure I fully understand the question

Reply:
I think the confusion comes with the naming of the class, because other RFC 0211 messages are called `MediationDeny` and `MediationGrant`, so the expected name for this one would be `MediationRequest`. From my point of view this naming would be consistent with the spec (which mentions them as 'Mediation Request', 'Mediation Deny', etc.).

The 'mediate-request' string within message type is OK, as it is the string defined in the spec so we should respect that (or modify the RFC **:-P**).

As for the Base URI, I guess it is fine, as long as it is the same defined in the RFC.

Reply:
https://github.com/hyperledger/aries-rfcs/tree/master/features/0211-route-coordination is an actual webpage while https://didcomm.org/coordinate-mediation is not. 
I agree with  'mediate-request' as the message type since the rfc is defined that way. I will update.